### PR TITLE
NEW: Add recipe for talisman

### PR DESCRIPTION
**Pull request by carterbox**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by carterbox** [comment]
@conda-forge/help-go, please review

**Review by xhochy — APPROVED** [review]

File: recipes/talisman/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  license_family: MIT
+  license_file:
+    - LICENSE
+    - license-files/
+  doc_url: https://thoughtworks.github.io/talisman/docs
+  dev_url: https://github.com/thoughtworks/talisman
+
+extra:
+  feedstock-name: talisman
+  recipemaintainers:
```
Comment (beckermr) [inline recipes/talisman/meta.yaml:51]: @carterbox @xhochy This line is wrong. IDK how the linter missed it, but we should fix in the feedstock.

**Comment by beckermr** [comment]
Ah this was merged when the linter was failing. We should not be doing that!

**Comment by xhochy** [comment]
I looked at the linter logs and the errors there were from a different recipe. So I attributed the linter failure here to general git confusion and not the recipe itself being broken :(

**Comment by xhochy** [comment]
Thank you for the feedstock fix!

**Comment by beckermr** [comment]
The logs are here: https://github.com/conda-forge/conda-forge-webservices/actions/runs/12202780198/job/34044254062#step:5:142 in the linked job.

**Comment by beckermr** [comment]
We should try and make it easier to find them for sure.

File: recipes/talisman/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  build:
+    - {{ compiler('go') }} 1.18
+    - go-licenses
+
+test:
+  commands:
+    - talisman --version
+
+
+about:
```
Comment (beckermr) [inline recipes/talisman/meta.yaml:30]: This is wrong too.

**Pull request merged**